Sleepy-Pi-Setup.sh

```diff
@@ -82,11 +82,20 @@ if grep -q 'enable_uart=1' /boot/config.txt; then
 else
     echo 'enable_uart=1' | sudo tee -a /boot/config.txt
 fi
-if grep -q 'core_freq=250' /boot/config.txt; then
-    echo 'The frequency of GPU processor core is set to 250MHz already - skipping'
-else
-    echo 'core_freq=250' | sudo tee -a /boot/config.txt
-fi
+if [ $RPi3 != true ]; then
+    if grep -q 'core_freq=400' /boot/config.txt; then
+        echo 'The frequency of GPU processor core is set to 400MHz already - skipping'
+    else
+        echo 'core_freq=400' | sudo tee -a /boot/config.txt
```

Comment:
This will add a new line but not edit the current line, right?

Reply:
Yes, it appends a newline to the end of the file.

Reply:
Shouldn't the value be updated? Maybe not easily done but I guess I needed to tell because I noted ;) Have a great day! Peace.

Reply:
Raspbian doesn't ship with any `core_freq` setting in  `/boot/config.txt`, so unless someone has already manually set a core frequency that should not be an issue.  But yes, checking for the key and updating its value is a more complete solution.

```
# For more options and information see
# http://rpf.io/configtxt
# Some settings may impact device functionality. See link above for details

# uncomment if you get no picture on HDMI for a default "safe" mode
#hdmi_safe=1

# uncomment this if your display has a black border of unused pixels visible
# and your display can output without overscan
#disable_overscan=1

# uncomment the following to adjust overscan. Use positive numbers if console
# goes off screen, and negative if there is too much border
#overscan_left=16
#overscan_right=16
#overscan_top=16
#overscan_bottom=16

# uncomment to force a console size. By default it will be display's size minus
# overscan.
#framebuffer_width=1280
#framebuffer_height=720

# uncomment if hdmi display is not detected and composite is being output
#hdmi_force_hotplug=1

# uncomment to force a specific HDMI mode (this will force VGA)
#hdmi_group=1
#hdmi_mode=1

# uncomment to force a HDMI mode rather than DVI. This can make audio work in
# DMT (computer monitor) modes
#hdmi_drive=2

# uncomment to increase signal to HDMI, if you have interference, blanking, or
# no display
#config_hdmi_boost=4

# uncomment for composite PAL
#sdtv_mode=2

#uncomment to overclock the arm. 700 MHz is the default.
#arm_freq=800

# Uncomment some or all of these to enable the optional hardware interfaces
dtparam=i2c_arm=on
#dtparam=i2s=on
#dtparam=spi=on

# Uncomment this to enable the lirc-rpi module
#dtoverlay=lirc-rpi

# Additional overlays and parameters are documented /boot/overlays/README

# Enable audio (loads snd_bcm2835)
dtparam=audio=on
start_x=1
gpu_mem=128
```

Reply:
I've altered the script to catch cases where core_freq is set to an incorrect value or is commented out.